test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
```

Comment:
Consider adding a docstring at the beginning of the file describing the purpose of this test file and what it's testing.

```suggestion
"""This file tests the functionality of the `func` function in `test1.py`."""
```